src/translations/translations_EN.json

```diff
@@ -86,16 +86,16 @@
     "contributor_helper_text" : "Others who have contributed to the resource but does not need to be credited in future usage of the resource",
     "creator_helper_aria_label": "Show creator explanation",
     "creator_helper_example": "Examples: lecturers, teachers, authors og creators of the resource can be submitted here",
-    "creator_helper_text": "The persons, companies, organizations who are the rights holders",
+    "creator_helper_text": "The person, company, or organization who is the rights holder",
     "description_helper_aria_label": "Show description explanation",
     "description_helper_example1": "A short video about the sympathetic nervous system's components and how it acts",
     "description_helper_example2": "Presentation about keyboard navigation and universal access. The lecture is aimed at frontend developers and addresses WCAG 2.1",
     "description_helper_example3": "Illustration sketch of the body's internal organs. The illustration shows the position of the organs in relation to each other",
     "description_helper_text1": "Provide a short description about your resource content",
     "description_helper_text2": "Description examples",
     "file_title_helper_aria_label": "Show file title explanation",
-    "file_title_helper_example1": "Example: \"flower\" is a better file title than \"img_23434.jpg\"",
-    "file_title_helper_example2": "Example: the filename \"thesis.pdf\" can be used as file title",
+    "file_title_helper_example1": "\"flower\" is a better file title than \"img_23434.jpg\"",
```

Comment:
kanskje noe litt mer konkret enn flower om vi skal gi forslag til gode titler. Fex "Orange blossoms on tree" ?  Og "img_1234" for å framheve enda mer at det er et eksempel